app/assets/stylesheets/cards.css.scss

```diff
@@ -0,0 +1,74 @@
+$flickr-pink: #ff2e92;
+$flickr-blue: #007bdc;
+
+$loader-size: 30px; 
+
+$loader-gutter: $loader-size/10;
+$loader-transform: $loader-size + $loader-gutter * 2;
+
+.loader {
+  border-radius: $loader-size/2;
```

Comment:
Properties should be ordered background, border, border-radius, height, margin, position, width